app/controllers/api/base_controller/renderer.rb

```diff
@@ -616,6 +616,18 @@ def determine_extra_cols(klass)
           attr_name.to_sym
         end
       end
+
+      def status_from_resource(resource)
+        return :ok unless resource.kind_of?(Hash)
```

Comment:
Can you comment on this line...I understand it only because I understand the issue, but a casual reader won't understand why a non-hash is atuomatically :ok

Reply:
we sure have a lot of ways we can render objects. I hope the documentation will serve well for people (including me) in the future